Cargo.toml

```diff
@@ -10,7 +10,6 @@ exclude = ["performance.png", "chart/**"]
 keywords = ["float"]
 license = "Apache-2.0 OR BSL-1.0"
 repository = "https://github.com/boa-dev/ryu-js"
-readme = "README.md"
```

Comment:
We remove the README?

Reply:
The readme field, if not specified if there exist a `README.md` it will choose it by default 

See: <https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field>